geoips/xarray_utils/data.py

```diff
@@ -687,11 +704,12 @@ def sector_xarrays(
 
         if is_dynamic_sector(area_def):
             LOG.info(
-                "  Trying to sector %s with dynamic time %s, data time %s to %s, %s points",
+                "  Trying to sector %s with dynamic time %s, data time %s to %s, ",
                 area_def.area_id,
                 area_def.sector_start_datetime,
                 xobj.start_datetime,
                 xobj.end_datetime,
+                "%s points",
                 xobj["latitude"].size,
```

Comment:
I think that needs to be the multiline string?  I don't think it can be s;lit up like that?  Or maybe it can?

Reply:
Made this a multiline string. We should be good on this branch unless you see anything else that needs changed.